composer.json

```diff
@@ -11,7 +11,7 @@
         "ext-curl": "*",
         "guzzlehttp/guzzle": "~4.2.3",
         "guzzlehttp/progress-subscriber": "~1.1.0",
-        "symfony/console": "~2.7.3",
+        "symfony/console": "~2.8.0@dev",
```

Comment:
This is required as the SymfonyStyle changed in 2.8, before this text was prefixed with `// `